tools/util.py

```diff
@@ -205,20 +205,15 @@ def rm_readonly(func, path, _):
 
 
 def build_mode(default="debug"):
```

Comment:
I don't think the `default` parameter is used anywhere at all, so maybe remove that as well?